ui/desktop/src/components/sessions/SessionListView.tsx

```diff
@@ -356,9 +360,7 @@ const SessionListView: React.FC<SessionListViewProps> = React.memo(
       // Update state immediately for optimistic UI
       setSessions((prevSessions) =>
         prevSessions.map((s) =>
-          s.id === sessionId
-            ? { ...s, metadata: { ...s.metadata, description: newDescription } }
-            : s
+          s.id === sessionId ? { ...s, metadata: { ...s, description: newDescription } } : s
```

Comment:
do we need all the props at both root and `metadata` of this object? it seems like how we've flattened out the data model a lot this object could become flattened as well

Reply:
oops. there is no more metadata, this is just a remnant that will set this useless property that we then don't display until we get confirmation from the server. nice catch!